### ISSUE TITLE
change network plotting settings 

### ISSUE THREAD

**Issue by crazyhottommy** [CFeregrino/scWGCNA#14]
Hi,
Thanks for this package.
scW.p.network(MmLimbE155.scWGCNA, module=1)

How can I change the node and label color/size?

It seems the color of the nodes is pre-determined.

**Comment by CFeregrino** [comment]
Hello, thank you for your interest in the package
You can use the options for the function GGally::ggnet2() in order to change the plot. I have never tried it, but in principle it should work.
Let me know how it goes!

**Comment by crazyhottommy** [comment]
yes, I tried `label.color = "red"` and others, and it did not change anything.

**Comment by Mr-Laurent** [comment]
Hey @crazyhottommy , I found how to do this : 
```scW.p.network(MmLimbE155.scWGCNA, module=1)+geom_text(aes(label=toupper(substr(label, 1, 10))), color = "white", fontface = "bold")```
will do the trick, it adds a label over the label (couldn't find how to do it directly) but it works. the geo_text will take the annotation, here the "label" that correspond to gene names, "1" and "10" means that it shows the 1st to the 10th characters of the label.
